app/controllers/posts_controller.rb

```diff
@@ -1,8 +1,9 @@
 class PostsController < ApplicationController
```

Comment:
- [ ] it would be better to add the authorization method  `load_and_authorize_resource` in your posts class so that it ensures to authorize

Reply:
resolve issue